addons/sale_timesheet/models/project_overview.py

```diff
@@ -383,9 +398,13 @@ def _plan_get_stat_button(self):
         # if only one project, add it in the context as default value
         tasks_domain = [('project_id', 'in', self.ids)]
         tasks_context = self.env.context
-        tasks_projects = self.env['project.task'].sudo().search(tasks_domain).mapped('project_id')
-        if len(tasks_projects) == 1:
-            tasks_context = {**tasks_context, 'default_project_id': tasks_projects.id}
+
+        # filter out all the projects that have no tasks
+        task_projects_ids = self.env['project.task'].read_group([('project_id', 'in', self.ids)], ['project_id'], ['project_id'])
+        task_projects_ids = [p['project_id'][0] for p in task_projects_ids]
+
+        if len(task_projects_ids) == 1:
```

Comment:
search_count ?

Reply:
Unfortunately, if `task_projects_ids` has only one cell, then we use its value :/